src/test/java/com/spotify/github/v3/checks/CheckRunOutputTest.java

```diff
@@ -0,0 +1,49 @@
+/*-
+ * -\-\-
+ * github-client
+ * --
+ * Copyright (C) 2016 - 2022 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.github.v3.checks;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.spotify.github.v3.checks.ImmutableCheckRunOutput.Builder;
+import org.junit.Test;
+
+public class CheckRunOutputTest {
+
+  private Builder builder() {
+    return ImmutableCheckRunOutput.builder();
+  }
+
+  @Test
+  public void allowsCreationWithinLimits() {
+    builder().build();
+    builder()
+        .text("t".repeat((64 * 1024) - 1)).summary("s".repeat((1024 * 64) - 1)).build();
```

Comment:
Let's put some single numbers instead of the math here? That makes it a lot more obvious on what's going on and allows it to be compared to the values in the documentation.

Reply:
Probably makes sense to also do that in the non-test code and link to the documentation.